docs/ensuring-repro/openscpca-nf/index.md

```diff
@@ -0,0 +1,23 @@
+# The OpenScPCA Nextflow workflow
+
+In addition to the `OpenScPCA-analysis` repository, the Data Lab maintains [a separate repository called `OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
+The `OpenScPCA-nf` repository holds a [Nextflow workflow](https://www.nextflow.io/) which we use to generate results from mature `OpenScPCA-analysis` modules.
```

Comment:
This felt choppy, so I combined it into one sentence.
```suggestion
In addition to the `OpenScPCA-analysis` repository, the Data Lab maintains a separate [`OpenScPCA-nf` repository](https://github.com/AlexsLemonade/OpenScPCA-nf) that holds a [Nextflow workflow](https://www.nextflow.io/) which we use to generate results from mature `OpenScPCA-analysis` modules.
```

Reply:
In another place, we used the phrase "official results". I don't love that phrase, but I like the sentiment of it. We should say something like that here?